src/controllers/sessionController.ts

```diff
@@ -112,9 +124,12 @@ export default class SessionController {
           );
         }),
         E.map(_ =>
-          ResponseSuccessJson({
+          ResponseSuccessJson<PublicSession>({
```

Comment:
ditto
```suggestion
          ResponseSuccessJson({
```

Reply:
Fixed in [9ce3eea](https://github.com/pagopa/io-backend/pull/993/commits/9ce3eea19c71a4b21bbfc1a2b91ad73b1a939808)